Felienne/hedy
# [BUG FIX] Quiz level 5 question 1 nl correctly renders again

**Description**
In this PR we fix question 1 for level 5 in the NL quiz. The content was empty due to containing both `option_text` and `code` where we only expect one to be there. Apparently the rendering breaks when this occurs. We remove the code lines in the multiple-choice options fixing this bug.

**Fixes**
This PR fixes #2583.

**How to test**
Make sure Hedy is set to dutch, navigate to level 5 and start the quiz. Verify that the first question is now rendered correctly.
